rte/mo_rte_lw.F90

```diff
@@ -220,7 +220,7 @@ function rte_lw(optical_props, top_at_1, &
     !
     ! Ensure values of tau, ssa, and g are reasonable if using scattering
     !
-    if(check_values) error_msg =  optical_props%validate()
+    error_msg =  optical_props%validate()
```

Comment:
Hi, is this a change you meant to make, @alexeedm ? Now the values of `optical_props` will always be checked for validity.

Reply:
Yeah, this is just for consistency. `validate()` does checks governed by both `chech_values` and `check_extents`, so when `check_values` is `.false.` you may still want to check the extents.